modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties

```diff
@@ -123,6 +124,8 @@ element-sets=Element Sets
 empty-list=No results were found.
 enable-experimental-interface=Enable Experimental Interface
 enable-experimental-interface-description=If checked, experimental UI features will be displayed on the form builder.
+enable-legacy-ddm-index-fields=Enable Legacy Dynamic Data Mapping Index Fields
+enable-legacy-ddm-index-fields-description=If checked, legacy ddm field names with structureId will be created in the index. It is necessary to execute a full reindex after changing this configuration.
```

Comment:
@jorgediaz-lr, do you think it's worth to mention `structureId` here? I believe this is a very technical information which could not be very clear to all end users. What do you think?